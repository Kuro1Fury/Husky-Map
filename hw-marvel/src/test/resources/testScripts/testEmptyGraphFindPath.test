# Test empty graph can output correct path

CreateGraph chars

FindPath chars n1 n2