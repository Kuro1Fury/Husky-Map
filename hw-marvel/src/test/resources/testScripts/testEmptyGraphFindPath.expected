# Test empty graph can output correct path

created graph chars

unknown: n1
unknown: n2